### PR TITLE
Spelling: ": Bigversal"

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

**Comment by paramat** [comment]
Maybe 'Install Mod' is intentional to act as a name that refers to something? Otherwise it just means 'Install mod' which reads as an instruction. :-1: for that change.
The other changes are ok.

File: builtin/mainmenu/modmgr.lua [hunk 1]
```diff
@@ -379,7 +379,7 @@ function modmgr.installmod(modfilename,basename)
 			local targetpath = core.get_modpath() .. DIR_DELIM .. targetfolder
 			core.copy_dir(basefolder.path,targetpath)
 		else
-			gamedata.errormessage = fgettext("Install Mod: unable to find real modname for: $1", modfilename)
+			gamedata.errormessage = fgettext("Install mod: Unable to find real modname for: $1", modfilename)
```
Comment (paramat) [inline builtin/mainmenu/modmgr.lua]: `mod name`?

**Comment by comradekingu** [comment]
@paramat I split the difference. It is "modpack", so mod-name isn't out of question.
"mod name" would result in horrendous compound word errors in translation.

**Comment by comradekingu** [comment]
@paramat You mean it could possibly be "Installation mod"?

**Comment by paramat** [comment]
Ok, let's see what others think.

File: builtin/mainmenu/modmgr.lua [hunk 2]
```diff
@@ -335,8 +335,8 @@ function modmgr.installmod(modfilename,basename)
 	local modpath = modmgr.extract(modfile)
 
 	if modpath == nil then
-		gamedata.errormessage = fgettext("Install Mod: file: \"$1\"", modfile.name) ..
-			fgettext("\nInstall Mod: unsupported filetype \"$1\" or broken archive", modfile.type)
+		gamedata.errormessage = fgettext("Install mod: File: \"$1\"", modfile.name) ..
```
Comment (paramat) [inline builtin/mainmenu/modmgr.lua]: 'Install Mod' to refer to the function 'installmod'.

File: builtin/mainmenu/modmgr.lua [hunk 3]
```diff
@@ -335,8 +335,8 @@ function modmgr.installmod(modfilename,basename)
...
+			fgettext("\nInstall mod: Unsupported filetype \"$1\" or broken archive", modfile.type)
```
Comment (paramat) [inline builtin/mainmenu/modmgr.lua]: 'Install Mod' to refer to the function 'installmod'.

File: builtin/mainmenu/modmgr.lua [hunk 4]
```diff
@@ -359,7 +359,7 @@ function modmgr.installmod(modfilename,basename)
 				gamedata.errormessage = fgettext("Failed to install $1 to $2", basename, targetpath)
 			end
 		else
-			gamedata.errormessage = fgettext("Install Mod: unable to find suitable foldername for modpack $1", modfilename)
+			gamedata.errormessage = fgettext("Install mod: Unable to find suitable folder name for modpack $1", modfilename)
```
Comment (paramat) [inline builtin/mainmenu/modmgr.lua]: 'Install Mod' to refer to the function 'installmod'.

File: builtin/mainmenu/modmgr.lua [hunk 5]
```diff
@@ -379,7 +379,7 @@ function modmgr.installmod(modfilename,basename)
 			local targetpath = core.get_modpath() .. DIR_DELIM .. targetfolder
 			core.copy_dir(basefolder.path,targetpath)
 		else
-			gamedata.errormessage = fgettext("Install Mod: unable to find real modname for: $1", modfilename)
+			gamedata.errormessage = fgettext("Install mod: Unable to find real mod-name for: $1", modfilename)
```
Comment (paramat) [inline builtin/mainmenu/modmgr.lua]: 'Install Mod' to refer to the function 'installmod'.
'mod name'.

**Comment by paramat** [comment]
Trivial so line comments can be fixed on merge.

**Comment by paramat** [comment]
No response for a month, can reopen on progress.

File: builtin/mainmenu/modmgr.lua (hunk 2 above)
Comment (paramat) [reply builtin/mainmenu/modmgr.lua]: It seems that these messages are referring to the function 'modmgr.installmod()' which is possibly why it is written 'Install Mod', so how this was seems fine, it doesn't need changing.

File: builtin/mainmenu/modmgr.lua (hunk 5 above)
Comment (paramat) [reply builtin/mainmenu/modmgr.lua]: 'mod-name' is incorrect English.

**Comment by paramat** [comment]
> so mod-name isn't out of question.

It is, correct English has priority over translation issues.

If you make the requested changes i'll merge this.

**Comment by paramat** [comment]
Needs rebase.

File: builtin/mainmenu/modmgr.lua [hunk 6]
```diff
@@ -379,7 +379,7 @@ function modmgr.installmod(modfilename,basename)
 			local targetpath = core.get_modpath() .. DIR_DELIM .. targetfolder
 			core.copy_dir(basefolder.path,targetpath)
 		else
-			gamedata.errormessage = fgettext("Install Mod: unable to find real modname for: $1", modfilename)
+			gamedata.errormessage = fgettext("Install Mod: Unable to find real modname for: $1", modfilename)
```
Comment (paramat) [inline builtin/mainmenu/modmgr.lua]: 'mod name'

**Comment by paramat** [comment]
If rebasing is difficult feel free to open a new PR for this.

**Comment by SmallJoker** [comment]
> If rebasing is difficult feel free to open a new PR for this.

It's like
```bash
git checkout THIS_BRANCH
git fetch upstream master
git rebase upstream/master
# Resolve conflicts
git status
# 'git add' all relevant files
git push --force
```

**Comment by paramat** [comment]
This PR will be re-opened if there is progress.

**Comment by paramat** [comment]
Replaced by d6f2a1c4b8ab13cee92e2041b3410fe3548e88e6

**Pull request closed without merging**